ActiveListExtensions/Utilities/QuickList.cs

```diff
@@ -50,8 +50,11 @@ public void Remove(int index)
 			if (index < 0 || index >= Count)
 				throw new ArgumentOutOfRangeException(nameof(index));
 
-			if (index < _count)
-				Array.Copy(_items, index + 1, _items, index, _count - index - 1);
+			if (index < _count - 1)
+			{
+				Array.Copy(_items, index + 1, _items, index, _count - index);
```

Comment:
But the change to remove -1 from the last argument doesn't look correct. If you have 6 items in a list, and are removing index 2, you should be copying 3 values (index 3, 4, and 5). Without the minus one, it'll copy 4 values won't it? Am I overlooking something?

Reply:
It needs to copy one extra (which should be assumed to be null). Otherwise in your example, (old) index 5 will now be referenced in both indexes 4 and 5, which will leak memory because 5 will never be cleaned up. Also, when `_count==1`, `_count - index - 1` will be `0` which won't move anything.

Reply:
Now that I think about it though, I think it will actually crash when `_count == _items.Length`. Good catch - I'll give it some more thought.

Reply:
@JohannesMoersch You were correct about the `- 1` still being needed. That change was left over from my first attempt at fixing this, and then I went in a different direction but forgot to remove it. I pushed a change to add that back, tested and it still appears to fix the memory leak.